src/infraestructure/db/repositories/event.repository.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import { MongoRepository } from './mongo.repository';
+import { Event } from '../../../domain/entities';
+import { Event as EventSchema } from '../schema';
+import { mapToEventsEntity } from '../services';
+import { DataBaseInternalError } from '../errors';
+
+@Injectable()
+export class EventReposiory extends MongoRepository<EventSchema> {
```

Comment:
You forgot the t here 
```suggestion
export class EventRepository extends MongoRepository<EventSchema> {
```